Analysis/analysis.r

```diff
@@ -1,7 +1,6 @@
 rm(list = ls())
 
 library(Hmisc)
-library(lubridate)
 library(hrbrthemes)
```

Comment:
It won't let me comment in the exact place as they are changed in this PR but looks like lines 18-30 (reading in 2019 data and merging with obtain weight target) is repeated a bit in the modelling scripts. Could this be 1 function that is then read into various scripts?